README.md

```diff
@@ -7,916 +7,53 @@
 
-  - [General model](#general-model)
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- [Layer](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)-based configuration
+- Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/) (aggregated events)
+- Identities
+- Can be used in machines and [containers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
+- [CLI commands](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/cli-commands/)
+- [HTTP API](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/) with a [Go client](https://pkg.go.dev/github.com/canonical/pebble/client) and a [Python client](https://github.com/canonical/operator/blob/main/ops/pebble.py)
 
-Pebble is organized as a single binary that works as a daemon and also as a
-client to itself. When the daemon runs it loads its own configuration from the
-`$PEBBLE` directory, as defined in the environment, and also records in
-that same directory its state and unix sockets for communication. If that variable
-is not defined, Pebble will attempt to look for its configuration from a default
-system-level setup at `/var/lib/pebble/default`. Using that directory is encouraged
-for whole-system setup such as when using Pebble to control services in a container.
+## Quick start
 
-The `$PEBBLE` directory must contain a `layers/` subdirectory that holds a stack of
-configuration files with names similar to `001-base-layer.yaml`, where the digits define
-the order of the layer and the following label uniquely identifies it. Each
-layer in the stack sits above the former one, and has the chance to improve or
-redefine the service configuration as desired.
+Prerequisites:
 
-## Layer configuration examples
+- A Linux machine.
 
-Below is an example of the current configuration format.
-For full details of all fields, see the [complete layer specification](#layer-specification).
-
-```yaml
-summary: Simple layer
-
-description: |
-    A better description for a simple layer.
+```bash
+go install github.com/canonical/pebble/cmd/pebble@latest
```

Comment:
We should figure out which installation approach we want to recommend, and use the same one as in the tutorial. On balance it's probably best to use standard Linux tools and not require Go to be installed, so IMO let's go with what's in the tutorial.

Perhaps we can just link to the installation doc to avoid repeating a bunch of stuff. Maybe:

> At any Linux shell, [install Pebble](https://canonical-pebble.readthedocs-hosted.com/en/latest/how-to/install-pebble/#how-to-install-pebble) and then run the following:
>
>     mkdir -p ~/.config/pebble/layers
>     ...

Reply:
Although using a link reduces duplicated content, it increases the barrier for people who want to give it a "quick start". The user journey would become:

Found the Pebble repo -> read the README -> click a link and jump to another tab -> copy-paste-run three commands -> switch back to the repo tab -> copy-paste the following command.

I think we want to reduce the friction to a minimum so that users can easily start without any hassle.

Even if we copy the three commands from the [how to install pebble](https://canonical-pebble.readthedocs-hosted.com/en/latest/how-to/install-pebble/#how-to-install-pebble) doc, users still need to replace the version tag.

I think `go install` is the simplest way, just one line of command, no need to replace anything, and it's OK to be used in the README because people who check out the repo are most likely developers.

It's also OK to use a different approach in the official doc because first of all, it's a more "official" way of installing a specific version; and second, the doc's audience includes normal users who are not developers.

So, I'd like to keep the `go install` here. Any objections? 

```bash

Reply:
Good reasoning -- you've convinced me, thanks. Let's keep `go install`.